app/lib/theme/widgets/ReadOnlyCalculatedValueWidget.tsx

```diff
@@ -0,0 +1,43 @@
+import { WidgetProps } from "@rjsf/core";
+import NumberFormat from "react-number-format";
+
+const ReadOnlyCalculatedValueWidget: React.FC<WidgetProps> = ({
+  id,
+  formContext,
+  label,
+  uiSchema,
+}) => {
+  // If we are using this widget to show numbers as money or percent, we can set `isMoney` or `isPercent` to true in the uiSchema.
+  const isMoney = uiSchema?.isMoney;
+  const isPercent = uiSchema?.isPercent;
+
+  return (
+    <dd>
+      {formContext.calculatedValue ? (
+        <NumberFormat
+          fixedDecimalScale={isMoney || isPercent}
+          prefix={isMoney ? "$" : ""}
+          suffix={isPercent ? "%" : ""}
+          id={id}
+          decimalScale={isMoney || isPercent ? 2 : 10} //Hardcoded for now, we can change it if we need to
+          value={formContext.calculatedValue}
+          displayType="text"
+          aria-label={label}
+        />
+      ) : (
+        <NumberFormat
+          fixedDecimalScale={isMoney || isPercent}
+          prefix={isMoney ? "$" : ""}
+          suffix={isPercent ? "%" : ""}
+          id={id}
+          decimalScale={isMoney || isPercent ? 2 : 10}
+          value={0}
+          displayType="text"
+          aria-label={label}
+        />
```

Comment:
If the calculated value is not set, this will look like we have a calculated value of "0" instead of something like "not available" (or just nothing at all, or an error message?)